py3status/modules/deadbeef.py

```diff
@@ -3,95 +3,101 @@
 Display song currently playing in deadbeef.
 
 Configuration parameters:
-    cache_timeout: how often we refresh usage in seconds (default 1)
-    delimiter: delimiter character for parsing (default '¥')
-    format: see placeholders below (default '{artist} - {title}')
+    cache_timeout: refresh interval for this module (default 1)
+    delimiter: delimiter between metadata fields (default '¥å¥å¥å')
+    format: display format for this module (default '{artist} - {title}')
 
 Format placeholders:
-    {artist} artist
-    {title} title
-    {elapsed} elapsed time
-    {length} total length
-    {year} year
-    {tracknum} track number
+    {artist}        artist
+    {length}        length
+    {playback_time} playback time
+    {title}         title
+    {tracknumber}   track number
+    {year}          year
 
 Color options:
-    color_bad: An error occurred
+    color_degraded: Paused
+    color_good: Playing
+    color_bad: Not Playing
 
 Requires:
-        deadbeef:
+    deadbeef: a GTK+ audio player for GNU/Linux
 
 @author mrt-prodz
 """
-from subprocess import check_output, CalledProcessError
+
+import subprocess
```

Comment:
iirc error codes for status.... and also, different versions. Similar to archlinux. I think they added `is_playing` metafield for `0.7.2`.